src/core/tracking.js

```diff
@@ -105,8 +105,23 @@ export function getAction(type) {
   }[type] || TRACKING_TYPE_INVALID;
 }
 
+export function doNotTrackEnabled({
+  _navigator = typeof navigator !== 'undefined' ? navigator : null,
+  _window = typeof window !== 'undefined' ? window : null,
+} = {}) {
+  if (!_navigator || !_window) {
+    return false;
+  }
+
+  const dnt = _navigator.doNotTrack || _window.doNotTrack;
```

Comment:
Yeah, but those versions don't really respect the spec either and enable it by default. I've stuck with newer browsers who use DNT as opt-in. They're also older versions than we generally target on AMO even (for CSS we target the past two versions).